core/capabilities/log_event_trigger/log_event_trigger.go

```diff
@@ -0,0 +1,99 @@
+package log_event_trigger
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-plugin"
+
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/log_event_trigger/trigger"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+)
+
+const (
+	serviceName = "LogEventTriggerCapability"
+)
+
+type LogEventServiceGRPC struct {
+	trigger capabilities.TriggerCapability
+	s       *loop.Server
+}
+
+func main() {
```

Comment:
@kidambisrinivas We conventionally put the main.gos in plugins/cmd

Reply:
Moved the main.go file to plugins/cmd